python/ray/rllib/optimizers/multi_gpu_optimizer.py

```diff
@@ -44,7 +45,9 @@ def _init(self,
         if not num_gpus:
             self.devices = ["/cpu:0"]
         else:
-            self.devices = ["/gpu:{}".format(i) for i in range(num_gpus)]
+            self.devices = [
+                "/gpu:{}".format(i) for i in range(int(math.ceil(num_gpus)))
```

Comment:
isn't numpy already imported?

Reply:
Better to avoid numpy if you can use standard libraries instead.